patientsearch/api.py

```diff
@@ -203,10 +204,19 @@ def external_search(resource_type, methods=["GET"]):
 @api_blueprint.route('/logout', methods=["GET"])
 def logout(methods=["GET"]):
     # TODO: is there a PHI safe 'id' for the user (in place of email)?
-    user_id = oidc.user_getfield('email')
-    current_app.logger.info(
-        "logout on request",
-        extra={'tags': ['logout'], 'user_id': user_id})
-    terminate_session()
+    try:
+        user_id = oidc.user_getfield('email')
+        current_app.logger.info(
+            "logout on request",
+            extra={'tags': ['logout'], 'user_id': user_id})
+        terminate_session()
```

Comment:
Unrelated, but could this POST to Keycloak be the reason we were regenerating a cookie at logout? I see `session.clear()` is called AFTER, so maybe not...

Reply:
`session.clear()` did not fix the symptom and does not appear to be required (once the session type is in place).  That was an early attempt, which i left in tact as it feels like a good idea.